Makefile

```diff
@@ -0,0 +1,11 @@
+SUBDIRS=kafka-base zookeeper kafka kafka-connect kafka-connect-s2i stunnel-base zookeeper-stunnel kafka-stunnel topic-operator-stunnel
```

Comment:
I guess that `kafka-init`, `user-operator`, `entity-operator` and `entity-operator-stunnel` will come later in another PR right because they are not in this repo yet ? Just asking to be sure ;)

Reply:
Exactly, I was going to add them to this file as I added their directories